QUEST_LV_0200.tsv

```diff
@@ -5242,18 +5242,18 @@ QUEST_LV_0200_20150323_005241	$Gitis told you that the workers at the settlement
 QUEST_LV_0200_20150323_005242	$The Pyromancer Master told you that she will definitely help you. Talk to the Pyromancer Master again.
 QUEST_LV_0200_20150323_005243	$Collect the samples of Pink Chupalukas at the settlement land
 QUEST_LV_0200_20150323_005244	{memo X}Pyromancer Master told you that she will definitely help you. First get some samples of Pink Chupalukas at the settlement land.
-QUEST_LV_0200_20150323_005245	$Hand it over to the Pyromancer Master
+QUEST_LV_0200_20150323_005245	$Bring the items to the Pyromancer Master
 QUEST_LV_0200_20150323_005246	$You've obtained the samples of Pink Chupalukas. Hand them over to the Pyromancer Master.
-QUEST_LV_0200_20150323_005247	Obtain %s from the settlement land
+QUEST_LV_0200_20150323_005247	Obtain %s from Ziuluti Vacant Lot
```

Comment:
Could be, you do find them in the Gitis Settlement Area, but the marker is at Ziuluti Vacant Lot, and the quests always tell you what Area already.